.github/CODEOWNERS

```diff
@@ -684,9 +702,10 @@
 
 
 # QA team.
-/core/tests/ @nithusha21
-/assets/ @nithusha21
-/data/ @nithusha21
+# TODO(#15381): Add @nithusha21 as owner after 2022-05-01, lines 706-708
+/core/tests/ @oppia/automated-qa-reviewers
+/assets/ @oppia/automated-qa-reviewers
```

Comment:
The Automated QA team isn't well prepared to own the `/assets/` folder

Reply:
Hi Chris, sorry I don't have access to my laptop now. Could you help move this to either Akshay or Kevin? I think I might have incorrectly assigned this to the automated QA team

Reply:
done